src/main/java/com/example/demo/service/study/CafeStudyService.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.service.study;
+
+import com.example.demo.dto.study.CafeStudyCreateRequest;
+import com.example.demo.dto.study.CafeStudyCreateResponse;
+
+public interface CafeStudyService {
+	// void tryJoin(long memberIdThatExpectedToJoin, long studyId);
+	//
+	// void tryQuit(long memberIdThatExpectedToQuit, long studyId);
+	//
+	// PagedResponse<StudyOnceSearchListResponse> searchStudy(StudyOnceSearchRequest studyOnceSearchRequest);
+	//
+	// StudyOnceSearchResponse searchByStudyId(long studyId);
+	//
+	// StudyOnceSearchResponse searchStudyOnceWithMemberParticipation(long studyOnceId, long memberId);
+	//
+	// UpdateAttendanceResponse updateAttendances(long leaderId, long studyOnceId,
+	// 	UpdateAttendanceRequest request, LocalDateTime now);
+	//
+	// void updateAttendance(long leaderId, long studyOnceId, long memberId, Attendance attendance, LocalDateTime now);
+
+	CafeStudyCreateResponse createStudy(long coordinatorId, CafeStudyCreateRequest cafeStudyCreateRequest);
```

Comment:
```suggestion
	Long createStudy(long coordinatorId, CafeStudyCreateRequest cafeStudyCreateRequest);
```
이부분에 대해서 논의 해보고 싶어요.
Long 타입으로 반환한 것은 CQS 개념과 관련되어 있는데, 문서 남겨드립니다.
https://martinfowler.com/bliki/CommandQuerySeparation.html
https://www.inflearn.com/community/questions/27795/cqrs

제가 경험해본바로는
DTO 를 반환했을 때
장점: 
1. 화면과 관련된 데이터를 반환하기때문에 바로 컨트롤러에 반환해서 응답에 넘겨주면 된다.

단점: 
1. UI 의존적이다.
3. UI에 의존적이기 떄문에 요구사항이 조금만 변경되더라도 DTO가 바뀐다. DTO가 바뀌면 테스트 코드에도 컴파일 오류가 발생한다. (다른말로는 리팩터링 내성이 낮아진다)

CQS에 맞춰서 void 인 경우
장점: 
1. Command와 Query를 완전히 분리하여 구별할 수 있다.

단점: 
1. 보통 CUD의 경우 HTTP응답으로 전달해줘야 하는 상황이 많은데, 반환 값이 없으면 다른 로직을 통해서 반환해야한다.

Long타입을 반환할 경우
장점: 
1. UI 의존적이지 않다.
2. 테스트 코드에 영향을 미치지 않는다. 테스트하기 좋다.
3. void에 비해 좀 더 편리하다. Long 값을 통해서 쉽게 조회할 수 있다.

단점: 
1. void와 동일하게 화면 의존적이지 않기 때문에 한번 더 쿼리를 날려야한다.

Reply:
현재 프로젝트의 경우 UI가 변경될 가능성이 높아 UI에 의존적이지 않게 Long 타입을 반환한다면 좋을 것 같습니다.
CQS패턴을 사용해보지 않아서 이번 프로젝트에  적용해보고 싶네요.

CQS 개념을 자세히 공부해보지 못해서 좋은 의견을 드리진 못하지만 추후 더 공부해오도록 하겠습니다!